api/src/pcapi/connectors/acceslibre.py

```diff
@@ -263,11 +263,52 @@ def extract_street_name(address: str | None = None, city: str | None = None, pos
     return ""
 
 
+def address_match(
+    acceslibre_address: str | None = None,
+    acceslibre_city: str | None = None,
+    acceslibre_postal_code: str | None = None,
+    acceslibre_ban_id: str | None = None,
+    venue_address: str | None = None,
+    venue_city: str | None = None,
+    venue_postal_code: str | None = None,
+    venue_ban_id: str | None = None,
+) -> bool:
+    # If banIds are matching, we return True
+    ban_ids_match = venue_ban_id and venue_ban_id == acceslibre_ban_id
```

Comment:
[détail / question de goût] Je pense qu'on gagnerait en lisibilité si on sortait prématurément de la fonction (et on n'aurait alors plus besoin du commentaire) : 

```suggestion
    if venue_ban_id and venue_ban_id == acceslibre_ban_id:
        return True
```

Reply:
Je prends, on va éviter la double négation